dtos/requests/operation.go

```diff
@@ -20,6 +19,7 @@ type OperationRequest struct {
 	dtoCommon.BaseRequest `json:",inline"`
 	ServiceName           string `json:"serviceName" validate:"required"`
 	Action                string `json:"action" validate:"oneof='start' 'stop' 'restart' 'rm' 'inspect'"`
+	UseRegistry           string `json:"useRegistry"` // to identify whether the app service container will be started with the --registry flag
```

Comment:
to be more flexible, add a field `flags []string` instead
it could be any flags and add to the container CMD
for the current case, SMA controller could add `--registry` to the flags if the registry is used

Reply:
Done